content/1_General/Running_Code_Locally.mdx

```diff
@@ -276,288 +270,76 @@ work with the latter two. This is one of the reasons why you should **not** use
 
-To get the full functionality of C++ on your Windows Machine, you should install
-one of the following compilers, from easiest to most difficult to set up and
-use:
-
-#### Simple: [MinGW](http://mingw.org/) (Minimalist GNU for Windows)
-
-If you want to keep the ease of use and simplicity of Linux C++, including the
-ability to use commands like `g++` and easily include `<bits/stdc++.h>`, you can
-install the compiler MinGW. This is a version of the `g++` compiler from Linux
-available on Windows. If you are using VS Code, setting up MinGW can be more
-difficult, but there are great guides to make the process easier.
-
-<Resources>
-	<Resource
-		source="MinGW"
-		title="MinGW Installation (Official)"
-		url="https://osdn.net/projects/mingw/downloads/68260/mingw-get-setup.exe"
-	>
-		The Official MinGW Installer
-	</Resource>
-	<Resource
-		source="Microsoft"
-		title="MinGW Installation (Easier)"
-		url="https://tinyurl.com/mingw-usaco"
-	>
-		Microsoft's Simpler MinGW Installer
-	</Resource>
-	<Resource
-		source="Microsoft"
-		title="MinGW with VS Code"
-		url="https://code.visualstudio.com/docs/cpp/config-mingw"
-	>
-		Setting Up MinGW with VS Code
-	</Resource>
-	<Resource
-		source="Jetbrains"
-		title="Configuring CLion on Windows"
-		url="https://www.jetbrains.com/help/clion/quick-tutorial-on-configuring-clion-on-windows.html#MinGW"
-	>
-		Setting up MinGW with CLion
-	</Resource>
-</Resources>
-
-Depending on the installer you used, installing MinGW could be a little
-complicated.
-
-**The Official Installer:**
-
-1.  First, download and run the
-    [MinGW installer](https://osdn.net/projects/mingw/downloads/68260/mingw-get-setup.exe/).
-2.  Once it's installed, open the MinGW Installation Manager, click on Basic
-    Setup on the left, and select `mingw32-gcc-g++-bin` for installation.
-3.  Finally, Add it to your PATH following the latter half of
-    [this tutorial](http://www.codebind.com/cprogramming/install-mingw-windows-10-gcc/)
-
-**Microsoft's Mirror Installer:**
-
-1.  First, [download the installer](https://tinyurl.com/mingw-usaco)
-2.  When you run it, leave all settings at default except `Architecture`.
-    1.  For `Architecture`, select `x86_64`.
-3.  After you click next, copy the `Destination Folder Path` for later and
-    finish the installation.
-4.  Now, you need to add MinGW to your `Path`. To do that:
-    1.  After the Installation has completed, search for
-        `Environmental Variables` in the Windows Search Bar and click open
-    2.  Click the button `Environment Variables` at the bottom left.
-    3.  In the bottom half of the screen (under `System variables`), find the
-        key `Path` and double click.
-    4.  Click `New` on the top right and paste the path you copied. Add `\bin`
-        to the end of it. It should look something like this:
-        `C:\Program Files\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin`
-    5.  You're Good to Go!
-
-#### Moderate: Microsoft Compiler Toolset
```

Comment:
Out of place - the other methods install some variant of `g++`. Is there a reason to prefer this method over the others? @Ta180m